lib/ansible/module_utils/rax.py

```diff
@@ -305,6 +307,8 @@ def setup_rax_module(module, rax_module, region_required=True):
     except KeyError as e:
         module.fail_json(msg='Unable to load %s' % e.message)
 
+    token = token or os.environ.get('OS_TOKEN')
+    tenant_name = tenant_name or os.environ.get('OS_PROJECT_NAME')
```

Comment:
These environment variable names should probably be `RAX_TOKEN` and `RAX_TENANT_NAME` to match the existing `RAX_` prefixed environment variables used by this module_util.

Also note the change from `PROJECT` to `TENANT` to match the terminology used in the API docs.